### DISCUSSION TITLE
Not implemented brokers

### DISCUSSION THREAD

**Discussion by s3rius** [taskiq-python/taskiq#85] [General]
In this discussion we decide which broker needs to be implemented. 

Currently I see these unimplemented brokers:
* [AWS SQS](https://aws.amazon.com/ru/sqs/);
* [Apache Kafka](https://kafka.apache.org/);
* [QPID](https://qpid.apache.org/) ( We need to check that aio-pika is compatible with it );
* [RedHAT AMQ](https://www.redhat.com/en/technologies/jboss-middleware/amq);
* [Apache ActiveMQ](https://activemq.apache.org/overview.html);
* [EMQX](https://www.emqx.com/en/products/emqx);
* [KubeMQ](https://kubemq.io/product-overview/);
* [NSQ](https://nsq.io/).

Also, some brokers may require additional tweaks to taskiq. We also try to decide it here.

**Comment by s3rius** [comment]
I would go with Apache kafka first. It must be relatively easy to implement in python and has a big community around it. @taskiq-python/core-team, what do you think?

**Reply by s3rius** [reply]
@chandr-andr implemented kafka lib: https://github.com/taskiq-python/taskiq-aio-kafka.

**Comment by fubuloubu** [comment]
We have implemented taskiq-sqs here: https://github.com/ApeWorX/taskiq-sqs

We are currently not maintaining it, do you want us to transfer it to you?

**Comment by danfimov** [comment]
Interesting, we have Azure Service Bus broker: https://github.com/hamiltonkibbe/taskiq-service-bus - [discussion](https://github.com/orgs/taskiq-python/discussions/592)
We should probably add link to docs.
